Projections/TestHost.ProbeByClass/TestHost.ProbeByClass.csproj

```diff
@@ -1,9 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <AssemblyName>TestHost.ProbeByClass</AssemblyName>
+    <TargetFrameworks>netstandard2.0;;net5.0</TargetFrameworks>
+    <Platforms>x64;x86</Platforms>
   </PropertyGroup>
 
-  <Import Project="../Test/Test.csproj"/>
+  <ItemGroup>
```

Comment:
Any reason why we decided to rebuild the test projection here rather than consume it like before?

Reply:
it was creating an msbuild warning